dataRetriever/resolvers/epochproviders/arithmeticEpochProvider_test.go

```diff
@@ -213,3 +214,22 @@ func TestArithmeticEpochProvider_NotifyOrder(t *testing.T) {
 
 	assert.Equal(t, uint32(core.CurrentNetworkEpochProvider), aep.NotifyOrder())
 }
+
+func TestEpochsDiff(t *testing.T) {
+	roundTimeInMilliseconds := uint64(5000)
+	roundsPerEpoch := 500
+	millisInASec := 1000
+
+	currentTimeStamp := uint64(1600000000)
+
+	epochDuration := uint64(roundsPerEpoch+1) * (roundTimeInMilliseconds / uint64(millisInASec))
+
+	headerTimestampForNewEpoch := currentTimeStamp - (epochDuration)
+
+	diffTimeStampInSeconds := currentTimeStamp - headerTimestampForNewEpoch
+	diffTimeStampInMilliseconds := diffTimeStampInSeconds * 1000
+	diffRounds := diffTimeStampInMilliseconds / roundTimeInMilliseconds
+	diffEpochs := diffRounds / uint64(roundsPerEpoch+1)
+
+	fmt.Println(diffEpochs)
```

Comment:
I see that you committed my "test" test. You can replace the print with an assert

Reply:
done